
Thread #1 is the program's root thread

Thread #2 was created
   at 0x........: clone (in /...libc...)
   by 0x........: ...
   by 0x........: pthread_create@GLIBC_ (in /lib/libpthread...)
   by 0x........: pthread_create@* (hg_intercepts.c:...)
   by 0x........: main (tc05_simple_race.c:27)

Possible data race during read of size 4 at 0x........ by thread #1
   at 0x........: main (tc05_simple_race.c:33)
 This conflicts with a previous access by thread #2
   at 0x........: child_fn (tc05_simple_race.c:19)
   by 0x........: mythread_wrapper (hg_intercepts.c:...)
   by 0x........: ...
   by 0x........: ...
 Location 0x........ is 0 bytes inside global var "y"
 declared at tc05_simple_race.c:10

ERROR SUMMARY: 1 errors from 1 contexts (suppressed: 0 from 0)
